app/Functions/GlobalFunctions.php

```diff
@@ -68,7 +68,7 @@ function calculateCacheTime($timeframe_type, $timeframe){
         if($timeframe[0] != getMaxGameVersion()){
           return 86400; //24 hours
         }else{
-          $date = getMaxGameVersionForGlobalReleaseDate();
+          $date = (string) getMaxGameVersionForGlobalReleaseDate();
```

Comment:
Why is it doing casting.  Is it just a preferred method of showing that the return needs to be a string or?

Reply:
Because `strtotime()` only accepts strings. During execution this is converted on-the-fly without issue (if strict mode is not enabled), but static analysis complains about it.